examples/gsutil/templates/startup-script-custom.tpl

```diff
@@ -0,0 +1,16 @@
+#! /bin/bash
+#
+# This is an example of how stdlib::get_from_bucket behaves.
+#
+# Check for crcmod
+# https://cloud.google.com/storage/docs/gsutil/addlhelp/CRC32CandInstallingcrcmod
+stdlib::cmd gsutil version -l
```

Comment:
What is this doing? Do we have a test assertion checking for the output from this command?

Reply:
Good catch, this touches on incomplete tests that were half written.

I've started cleaning up the test cases and will continue, but probably won't finish it before the weekend.

Tests to add without any other cruft:

 * [ ] Expect gsutil version -l to report `compiled crcmod: False` prior to calling stdlib::init_gsutil_crcmod_el
 * [ ] Expect `compiled crcmod: True` after calling stdlib::init_gsutil_crcmod_el
 * [ ] Expect stdlib::get_from_bucket to download message.txt when the instance has `storage.objects.get` access.
 * [ ] Expect stdlib::get_from_bucket to retry up to 10 times when the instance does not have `storage.objects.get` access